source/includes/orr-intro.rst

```diff
@@ -0,0 +1,3 @@
+Starting in {+c2c-beta-program-short+} 1.8, you can enable Oplog Rollover 
```

Comment:
(consistency)
Here, mongosync is in monospace, while beta is not. Above, in Filter Documents mongosync-beta is in monospace, and has a hyphen. I think you may just need to make Filter Documents consistent with everything else here. 

Preview link for convience: https://preview-mongodbajhuhmdb.gatsbyjs.io/cluster-sync/DOCSP-38286-enable-ORR/reference/beta-program/#beta-features

Reply:
Ah gotcha, thanks for catching that. That text in the filter documents page was added before we created a snooty constant for it. Fixed!

Reply:
Delightful! TY!